openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmRemoteMemoryNoAuth.java

```diff
@@ -121,7 +123,8 @@ public void execute(StfCoreExtension test) throws StfException {
 		dumpFile = resultsDir.childFile("javacore_mem_server.%Y%m%d.%H%M%S.%pid.%seq.txt,filter=java.lang.IllegalArgumentException");
 		
         JavaProcessDefinition clientJavaInvocationServer =  test.createJavaProcessDefinition()
-		    .addJvmOptionIfIBMJava("-Xdump:java:events=throw,file=" + dumpFile.getSpec())
+    		.addJvmOption("-Xmx256m")
+        	.addJvmOptionIfIBMJava("-Xdump:java:events=throw,file=" + dumpFile.getSpec())
```

Comment:
Same with comment in TestJlmGCMXBeanNotifier.java, you may want to replace space back with tab.

Reply:
Thanks! Corrected it.